### ISSUE TITLE
question about data augmentation

### ISSUE THREAD

**Issue by jasonseu** [Alibaba-MIIL/ASL#77]
https://github.com/Alibaba-MIIL/ASL/blob/68166d5e81301f3d936f45e6fd3615e267e33ce7/train.py#L59

As far as I know, transforms.ToTensor does not have normalization.

**Comment by mrT23** [comment]
Have you checked in PyTorch docs ?
https://pytorch.org/vision/master/generated/torchvision.transforms.ToTensor.html

**Comment by jasonseu** [comment]
> 

Yes, I have read the docs. ToTensor in PyTorch only Converts the scale of the input image data from the range [0, 255] to the range [0.0, 0.1], while normalization is achieved by transforms.Normalize, which normalizes a tensor image (usually the output of transforms.ToTensor) with mean and standard deviation.

**Comment by jasonseu** [comment]
> Have you checked in PyTorch docs ? https://pytorch.org/vision/master/generated/torchvision.transforms.ToTensor.html

Yes, I have read the docs. ToTensor in PyTorch only Converts the scale of the input image data from the range [0, 255] to the range [0.0, 0.1], while normalization is achieved by transforms.Normalize, which normalizes a tensor image (usually the output of transforms.ToTensor) with mean and standard deviation.

**Comment by mrT23** [comment]
read the article appendix B:
"
We found that the
common ImageNet statistics normalization [14, 8, 27] does
not improve results, and instead used a simpler normalization - scaling all the RGB channels to be between 0 an 1
"

a tip for the future - don't follow blindly "common practices" that do not make sense. test them yourself
